addons/portal/controllers/portal.py

```diff
@@ -250,6 +248,8 @@ def _update_password(self, old, new1, new2):
         except UserError as e:
             return {'errors': {'password': e.name}}
         except AccessDenied as e:
+            # FIXME XMO dead code since https://github.com/odoo/odoo/pull/45723
+            # AccessDenied is an UserError now
             msg = e.args[0]
             if msg == AccessDenied().args[0]:
                 msg = _('The old password you provided is incorrect, your password was not changed.')
```

Comment:
cc: @xmo-odoo

Reply:
Yes?

Reply:
Dunno if you want to keep this logic with the specific message in case of AccessDenied, but it's dead code in master currently.